src/pages.cpp

```diff
@@ -2280,7 +2384,7 @@ static void add_option_widget(GtkWidget *widget, Option *o)
         break;
 
       case OPTION_TYPE_DYNAMIC_LIST:
-        if (std::string(o->name) == "command")
+        if (std::string(o->name) == "bindings")
```

Comment:
Same question here, why is this being changed?

Reply:
Because the XML was also renamed. Now we have three different groups, `bindings`, `repeatable_bindings` and `always_bindings`. They don't need separate handling in WCM however (it all works out when rebuilding the values with the prefixes).